### ISSUE TITLE
Bug: Rules That Don't Load Should Return an Error

### ISSUE THREAD

**Issue by jacknagz** [panther-labs/panther_analysis_tool#29]
When running a rule that throws an exception, the test summary should show a failure:

```
[WARNING]: Unexpected exception: {An error occurred (AccessDenied) when calling the GetObject operation: Access Denied}
	[PASS] Description to other test
--------------------------
Panther CLI Test Summary
	Path: /tmp/l8h0e7GNot
	Passed: 113
	Failed: 0
	Invalid: 0
```
